<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configPage</class>
 <widget class="QWidget" name="configPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config Page</string>
  </property>
  <widget class="QPushButton" name="changeDirButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Dir</string>
   </property>
  </widget>
  <widget class="QLabel" name="compressLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>501</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Is Compressed?</string>
   </property>
  </widget>
  <widget class="QLabel" name="encryptionLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>501</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dirLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Back-up Dir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Is Encryption?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
